green-ev/app/dashboard/map/page.tsx

```
@@ -0,0 +1,111 @@
+import { chargingStationApi } from '@/lib/api'
+import { Card, CardContent } from '@/components/ui/card'
+import 'leaflet/dist/leaflet.css'
+
+// Fix for default marker icon
+import L from 'leaflet'
+import icon from 'leaflet/dist/images/marker-icon.png'
+import iconShadow from 'leaflet/dist/images/marker-shadow.png'
+
+// Fix default marker icons (prevents broken images)
+if (typeof window !== 'undefined') {
+  delete (L.Icon.Default.prototype as any)._getIconUrl;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)